doc/source/whatsnew/v2.0.0.rst

```diff
@@ -28,13 +28,16 @@ The available extras, found in the :ref:`installation guide<install.dependencies
 ``[all, performance, computation, timezone, fss, aws, gcp, excel, parquet, feather, hdf5, spss, postgresql, mysql,
 sql-other, html, xml, plot, output_formatting, clipboard, compression, test]`` (:issue:`39164`).
 
-.. _whatsnew_200.enhancements.io_readers_nullable_pyarrow:
+.. _whatsnew_200.enhancements.io_use_nullable_dtypes_and_nullable_backend:
 
 Configuration option, ``io.nullable_backend``, to return pyarrow-backed dtypes from IO functions
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-A new global configuration, ``io.nullable_backend`` can now be used in conjunction with the parameter ``use_nullable_dtypes=True`` in :func:`read_parquet`, :func:`read_orc` and :func:`read_csv` (with ``engine="pyarrow"``)
-to return pyarrow-backed dtypes when set to ``"pyarrow"`` (:issue:`48957`).
+The ``use_nullable_dtypes`` keyword argument has been expanded to :func:`read_csv` and :func:`read_excel` to enable automatic conversion to nullable dtypes (:issue:`36712`)
```

Comment:
cc @phofl this is the section where we can expand on `use_nullable_dtype`. Let me know if I should expand on anything in this PR

Reply:
I was thinking about adding a really short example with only one or 2 columns for read_csv

Reply:
Should I expand on the example below (it's hidden in the diff) to show `pd.option_context("io.nullable_backend", "pandas")` as well?

Reply:
Ah sorry, missed that. Yes a pandas example would be great.

Another thing: Maybe make the functions bullet points, e.g.

- read_csv
- read_excel

So that they become a bit more prominent?

Reply:
Good idea with the bullet points. Also added an example with the pandas example